lld/ELF/Relocations.h

```diff
@@ -92,6 +92,10 @@ enum RelExpr {
   R_AARCH64_PAGE_PC,
   R_AARCH64_RELAX_TLS_GD_TO_IE_PAGE_PC,
   R_AARCH64_TLSDESC_PAGE,
+  R_AARCH64_AUTH_TLSDESC_PAGE,
+  // TODO: maybe it's better to rename this expression
+  // to avoid name conflict with dynamic reloc
+  R_AARCH64_AUTH_TLSDESC,
```

Comment:
Could be `R_AARCH64_AUTH_TLSDESC_STATIC`?

Reply:
It's fine for me, but it would be glad to see @MaskRay 's opinion on rel exprs naming conventions :)

Reply:
Apologies for the slow response.
I propose that we rename these RelExpr enumerators to `REL_*`
https://github.com/llvm/llvm-project/pull/118424

Reply:
> Apologies for the slow response. I propose that we rename these RelExpr enumerators to `REL_*` #118424

Thanks @MaskRay! Updated #113815, #113816, #113817 to follow this naming convention